docs/examples/.eslintrc

```diff
@@ -31,6 +31,7 @@
     "Label",
     "ListGroup",
     "ListGroupItem",
+    "ListGroupItemHeading",
```

Comment:
Added `ListGroupItemText`, so its now up to the user to decide to use `< ListGroupItemText />` or use plain divs.